pages/pix-site/index.vue

```diff
@@ -1,63 +1,31 @@
-<template>
-  <div class="index">
-    <prismic-custom-slice-zone :slices="slices" />
-  </div>
-</template>
+<template><div></div></template>
 
 <script>
-import { documentFetcher } from '~/services/document-fetcher'
+import { language } from '~/config/language'
 
 export default {
-  name: 'Index',
-  nuxtI18n: {
-    paths: {
-      fr: '/',
-      'fr-fr': '/',
-      'en-gb': '/',
-      'fr-be': '/',
-    },
+  nuxtI18n: false,
+  layout: 'simple',
+  mounted() {
+    const chosenLocale = this.getLocaleFromCookie()
+    const url = chosenLocale ? `/${chosenLocale}/` : '/locale-choice'
+    this.$router.push(url)
```

Comment:
Le `push` ajoute une entrée dans l'historique de l'utilisateur, ce qui fait qu'on ne peut pas utiliser le bouton précédent du navigateur qui nous redirige automatiquement sur la page courante.

Je propose de remplacer par :

```suggestion
    this.$router.replace(url)
```

Pour ne pas sauvegarder l'accès au `/` dans l'historique. (dans une autre PR)